testing/resources/lua_repo/hello-dev-1.rockspec

```diff
@@ -0,0 +1,15 @@
+package = "hello"
+version = "dev-1"
+
+source = {
+   url = "git+ssh://git@github.com/pre-commit/pre-commit.git"
```

Comment:
I noticed the confused expression on your face on the stream when you got to this line. `luarocks` needs a URL. The value doesn't really matter for the operation of the `build` command, so I just opted to use something related to pre-commit. I could probably put something random here if you'd prefer.

Reply:
this is fine, it's just test data anyway